client.go

```diff
@@ -9,7 +9,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/dgrijalva/jwt-go"
+	"github.com/dgrijalva/jwt-go/v4"
```

Comment:
just one worry about `v4` that it is not officially released, but it great if everything works fine

Reply:
Yeah, people are waiting for the "stable release" but it works :shrug: 
https://github.com/dgrijalva/jwt-go/issues/348

Reply:
yeah, I've seen it and also wait for stable release. Unfortunately I cannot update to unstable